client/app/components/packages/pas-form/edit.hbs

```diff
@@ -130,7 +130,7 @@
               <RadioButton
                 @value={{radio.code}}
                 @groupValue={{this.saveableChanges.dcpUrbanrenewalarea}}
-                @changed={{fn this.validate}}
+                @changed={{this.validate}}
```

Comment:
Ah, so you're removing this because `fn` is only for when you need to apply other arguments to the function call? Plz explain. Thx!

Reply:
Yes, that's my understanding of `fn` ! It's for "binding" a function to arguments from a certain context. 
http://api.emberjs.com/ember/3.11/classes/Ember.Templates.helpers/methods/fn?anchor=fn